### ISSUE TITLE
[no-var] How to limit this rule to be inside of a class?

### ISSUE THREAD

**Issue by rgelb** [typescript-eslint/typescript-eslint#2231]
**Repro**

Consider the following code:

```TS
declare var Circles: any;  // reference a JavaScript library

class DoStuffer {
  public DoStuff(): void {
    var localVariable = 5;

   Circles.paint();
  }
}
```
along with a snippet from the following .eslintrc

```JSON
{
	"parser": "@typescript-eslint/parser",
	"ecmaFeatures": {"jsx": true},
	"plugins": ["@typescript-eslint"],
	"rules": {"no-var": 2}
}
```

**Expected Result**
The `no-var` rule ensures that there aren't any var declaration of variables (should be let or const).
It should not mark `declare var Circles: any;` an error because it's a TypeScript method to reference external JavaScript libraries.  It is not a traditional variable declaration.

**Actual Result**
It marks `declare var Circles: any;`  to be an error.

**Additional Info**
It correctly identified `var localVariable = 5;` as the error.  It seems that rule blanket-marks anything with a `var` to be an error.

**Versions**

| package                            | version |
| ---------------------------------- | ------- |
| `@typescript-eslint/eslint-plugin` | `3.3.0` |
| `@typescript-eslint/parser`        | `3.3.0` |
| `TypeScript`                       | `3.9.5` |
| `ESLint`                           | `7.3.0` |
| `node`                             | `12.16.3` |
| `npm`                              | `6.14.4` |

**Comment by bradzacher** [comment]
you can't.
you can use an [eslint disable comment](https://eslint.org/docs/user-guide/configuring#disabling-rules-with-inline-comments) to disable it for that line.

Why not use `declare const Circle` instead?

**Comment by rgelb** [comment]
@bradzacher You are absolutely correct.  All the TS documentation says to use var to reference a JS lib.  Didn't realize const was also possible.
